test/e2e/framework/node/resource.go

```diff
@@ -431,7 +412,9 @@ func isNodeUntaintedWithNonblocking(node *v1.Node, nonblockingTaints string) boo
 		}
 		n = nodeCopy
 	}
-	return toleratesTaintsWithNoScheduleNoExecuteEffects(n.Spec.Taints, fakePod.Spec.Tolerations)
+
+	var tolerations []v1.Toleration
+	return toleratesTaintsWithNoScheduleNoExecuteEffects(n.Spec.Taints, tolerations)
```

Comment:
```suggestion
	return toleratesTaintsWithNoScheduleNoExecuteEffects(n.Spec.Taints, nil)
```

Reply:
Thanks, merged, squashed, made you co-author.